starknet_py/tests/e2e/block_test.py

```diff
@@ -36,17 +15,17 @@ async def declare_contract(account_client: AccountClient):
 
 
 @pytest.mark.asyncio
-async def test_pending_block(new_gateway_account_client):
+async def test_pending_block(new_gateway_account_client, map_compiled_contract):
     # TODO: change to new_account_client once devnet repaired
```

Comment:
Is devnet repaired now? :hmm:

Reply:
It must be 🤔 And I think `new_gateway_account_client` should be changed to `account` to use a Account after refactor. Will address it in separate PR.